client/main.css

```diff
@@ -74,6 +78,10 @@
   line-height: 0.7;
 }
 
+.lh-terms {
```

Comment:
This is equivalent to `.lh-copy` (http://tachyons.io/docs/typography/line-height/), so should be removed